data/en/stocks_0001.json

```diff
@@ -504,9 +504,9 @@
                                             "value": "Change of business structure, merger or takeover"
                                         },
                                         {
-                                            "label": "One off increase in stocks",
+                                            "label": "One-off increase in stocks",
                                             "q_code": "146f",
-                                            "value": "One off increase in stocks"
+                                            "value": "One-off increase in stocks"
```

Comment:
Do they want the value changing as well? As I can see this is different in the dump submission

Reply:
🤷‍♂

Reply:
I assume it's okay as long as the q-codes remain the same.